src/main/java/Mysql/DOMParser.java

```diff
@@ -0,0 +1,273 @@
+package Mysql;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
+
+public class DOMParser {
+
+    public void customersDomParser(String XMLFilePath) {
+        try {
+
```

Comment:
As I remember parser/validator tasks should be committed in a different branch and PR based on how Vladislav explained

Reply:
Done!